src/js/index.js

```diff
@@ -0,0 +1,76 @@
+const total = document.querySelector('#total');
+const modifiers = document.querySelector('.modifiers');
+const operations = document.querySelector('.operations');
+const digits = document.querySelector('.digits');
+let prevDigit = 0;
+let result = [];
+const operator = ['+', '-', 'X', '/'];
+
+function calculators() {
+    modifiers.addEventListener('click', reset);
+    digits.addEventListener('click', insertDigit);
+    operations.addEventListener('click', insertOperator);
+}
+
+function reset() {
+    total.innerText = 0;
+    prevDigit = 0;
+    result = [];
+}
+
+// 숫자 입력
+function insertDigit(e) {
+    if (!prevDigit) total.innerText = 0;
+    if (prevDigit.toString().length > 2) return alert('숫자는 한번에 최대 3자리 수까지 입력할 수 있습니다.');
+
+    total.innerText = Number(total.innerText) === 0 ? e.target.innerText : total.innerText + e.target.innerText;
+    prevDigit = total.innerText;
+}
```

Comment:
```suggestion
function insertDigit(e) {
    if (prevDigit.toString().length > 2) return alert('숫자는 한번에 최대 3자리 수까지 입력할 수 있습니다.');

    total.innerText = Number(total.innerText + e.target.innerText);
    prevDigit = total.innerText;
}
```
<img width="143" alt="스크린샷 2022-11-07 오후 10 45 17" src="https://user-images.githubusercontent.com/18749057/200325577-153a3294-f1f4-45fb-b13e-d3ab1efb4456.png">
이렇게 표현할 수 있을 것 같아요!